src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html

```diff
@@ -4,35 +4,41 @@
 <!-------------------------->
 <div class="company-select-box">
   <div class="wrapper">
-    <div class="box-header">
-      <span class="title">{{ selectBox.user.name }} {{ selectBox.user.surname }}</span>
-      -
-      <span class="title">{{ selectBox.organization.name }}</span>
-      <a type="button" class="btn switch-company" ng-click="selectBox.toggle()">
-        <i ng-class="!selectBox.isClosed && 'active'"></i>
-      </a>
+    <div class="box-header" ng-click="selectBox.toggle()">
+      <img ng-src="{{selectBox.user.avatar_url}}" src="{{selectBox.user.avatar_url}}" />
```

Comment:
You don't need the `src` only the `ng-src`.

https://docs.angularjs.org/api/ng/directive/ngSrc